mkdocs/docs/HPC/running_batch_jobs.md

```diff
@@ -803,6 +803,29 @@ The output of the various commands interacting with jobs (`qsub`,
 `stat`, ...) all depend on which `cluster` module is loaded.
 {% endif %}
 
+### Submitting jobs from one cluster to another
+
+It is possible to submit jobs from a job to a cluster different than the one your job is running on.
+This could come in handy if, for example, the submit command only works on the current clusters worker nodes, but the job itself can run on other clusters. An example of this is the `wsub` command of `worker`. More technical info why the `wsub` command can only be executed on the {{defaultcluster}} cluster can be found [here](troubleshooting.md#multi-job-submissions-on-a-non-default-cluster).
+
+To submit jobs to the `{{othercluster}}` cluster, you can only change what is needed in your session environment to submit jobs to that particular cluster by using `module swap env/slurm/{{othercluster}}` instead of using `module swap cluster/{{othercluster}}`.
+The last command also activates the modules that are installed specifically for {{othercluster}}, and which may not be compatible with the system you are working on.
+That way, jobs that are submitted will be sent to the `{{othercluster}}` cluster. The same approach can be used to submit jobs to another cluster, of course.
+
+
+Each `cluster` module not only loads the corresponding `env/slurm/...` module to control where jobs are sent to, but also to other `env/...` modules which control other parts of the environment.
+For example, for the `{{defaultcluster}}` cluster:
+
+We did not only create the `env/slurm/{{defaultcluster}}` part, but we actually split every cluster in 3 different parts. Here illustrated with the `{{defaultcluster}}` cluster, but this can be changed to every cluster you like.
+
+| `env/` module for `{{defaultcluster}} | Purpose |
+|---------------------------------------|----------------|
+| `env/slurm/{{defaultcluster}}`        | Changes the cluster where jobs are sent to. |
+| `env/software/{{defaultcluster}}`     | Changes `$MODULEPATH`, which controls what software modules are available for loading. |
+| `env/vsc/{{defaultcluster}}`          | Changes the set of `$VSC_` environment variables that are specific to the `{{defaultcluster}}` cluster |
+
+Although we suggest that you don't use the seperate parts if you don't really need to or don't exactly know what you are doing, since using these seperetly can lead to some weird behaviour if used incorrectly.
```

Comment:
```suggestion
We recommend that you do not use these separate `env/` modules directly unless you really need to,
and only if you understand what they are doing exactly,
since mixing `cluster/` and `env/` modules of different clusters can result in surprises if you are not careful.